src/anyio/_core/_subprocesses.py

```diff
@@ -82,9 +85,12 @@ async def drain_stream(stream: AsyncIterable[bytes], index: int) -> None:
 
         stream_contents[index] = buffer.getvalue()
 
+    if stdin is not None and input is not None:
+        raise ValueError("only one of stdin and input is allowed")
```

Comment:
you probably want a test case for this.
(maybe look over coverage config to enforce 100% patch coverage?)

Reply:
Path coverage minimum set to 100% on Coveralls. Test case added.